components/blitz/src/ome/formats/OMEROMetadataStoreClient.java

```diff
@@ -1592,7 +1592,8 @@ public void setPixelsFile(long pixelsId, String file, String repo) throws Server
         catch (Exception e)
         {
             log.error("Server error setting extended properties for Pixels:" +
-                      pixelsId + " Target file:" + file);
+                      pixelsId + " Target file:" + file, e);
+            throw new RuntimeException(e);
```

Comment:
maybe something like,
```java
throw e instanceof RuntimeException ? (RuntimeException) e : new RuntimeException(e);
```
to avoid unnecessary wrapping?